### DISCUSSION TITLE
Import isosurface mesh

### DISCUSSION THREAD

**Discussion by bergwerf** [molstar/molstar#1383] [Ideas]
JSmol can compute and export an isosurface. Instead of being a full density map this is just a mesh with coloring data, resulting in much smaller files. Could we somehow import this into Mol*? I would appreciate tips or pointers on how to implement this!

**Comment by arose** [comment]
An option could be to export a mesh and than import it. Unfortunately molstar only supports importing PLY files but can only export glTF, STL, OBJ, and USD. Easiest is probably to add a PLY exporter. See `src\examples\glb-export\index.ts` for a programatic exporter example.
